xml/System.Threading/Timer.xml

```diff
@@ -594,7 +594,7 @@ Sub TimerCallback(state As Object)
         <Parameter Name="period" Type="System.Int64" Index="1" FrameworkAlternate="netcore-2.0;netcore-2.1;netcore-2.2;netcore-3.0;netframework-1.1;netframework-2.0;netframework-3.0;netframework-3.5;netframework-4.0;netframework-4.5;netframework-4.5.1;netframework-4.5.2;netframework-4.6;netframework-4.6.1;netframework-4.6.2;netframework-4.7;netframework-4.7.1;netframework-4.7.2;netframework-4.8;netstandard-2.0;xamarinandroid-7.1;xamarinios-10.8;xamarinmac-3.0;netstandard-2.1;netcore-3.1" />
       </Parameters>
       <Docs>
-        <param name="dueTime">The amount of time to delay before the invoking the callback method specified when the <see cref="T:System.Threading.Timer" /> was constructed, in milliseconds. Specify <see cref="F:System.Threading.Timeout.Infinite" /> to prevent the timer from restarting. Specify zero (0) to restart the timer immediately.</param>
+        <param name="dueTime">The amount of time to delay before the invoking the callback method specified when the <see cref="T:System.Threading.Timer" /> was constructed, in milliseconds. Specify <see cref="F:System.Threading.Timeout.Infinite" /> to prevent the timer from restarting. Specify zero (0) to restart the timer immediately.  This value must be less than or equal to 4294967294.</param>
```

Comment:
If we fix the exception being thrown, do you think it's still useful to mention the max value here?

Reply:
I think its still useful to mention the max value in the "dueTime" parameter documentation here.  It would probably be good to add the same information on the max value when describing the "period" parameter as well.  Not everyone reads all the documentation on the exceptions.  I think it makes sense to have information on maximum values when you are describing the parameters.  I also looked at your other suggestions above, and they look fine.

Reply:
Sounds good. I agree we should do the same for both parameters then. Can you please apply my suggestions above using the **Commit suggestion** button?

Reply:
Yes, I will commit the suggestions now.  Wasn't sure if I was supposed to do that as part of the process.